elasticsearch/src/main/scala/com/snowplowanalytics/stream/loader/clients/ElasticsearchBulkSender.scala

```diff
@@ -210,7 +210,7 @@ class ElasticsearchBulkSender(
     error: Option[String],
     record: EmitterJsonInput
   ): Option[EmitterJsonInput] = {
-    error.foreach(e => log.error("Failed to index record in Elasticsearch", e))
+    error.foreach(e => log.error(s"Failed to index record in Elasticsearch: {}", e))
```

Comment:
Actually, it is valid to do `log.error("message", throwable)` _if the second parameter is a throwable_.  Because [the method is overloaded to allow a throwable in the second position](https://www.slf4j.org/api/org/slf4j/Logger.html#error-java.lang.String-java.lang.Throwable-).

It's only if the second parameter is _not_ a throwable, that you need to use the template syntax in the message.

Reply:
I see, thanks for the clarification!